### PR TITLE
Add instructions for configuring custom domain for status page

### PR DESCRIPTION
**Pull request by amir-hadzic**
This PR adds instruction for how to configure custom domains for status pages.

### CODE REVIEW

File: docs/synthetics/status-pages.md [hunk 1]
```diff
@@ -26,6 +26,20 @@ Every status page will have a unique public URL of the format - `https://apps.se
 
 Disabling the status page will block the public access for the status page.
 
+# Custom Domain Setup
+
+To use a custom domain for your status page, eg. status.company.com, you will need to:
```
Comment (otisg) [inline docs/synthetics/status-pages.md:31]: eg. => e.g.,

File: docs/synthetics/status-pages.md [hunk 2]
```diff
@@ -26,6 +26,20 @@ Every status page will have a unique public URL of the format - `https://apps.se
...
+
+ * enter the domain in the Custom domain field
+ * configure your DNS provider
+
+To configure your DNS provider, add a CNAME record for `status.company.com`. The record should point to:
+
+ * `statuspage.us.sematext.com` if your App is created in the Sematext Cloud US region
+ * `statuspage.eu.sematext.com` if your App is created in the Sematext Cloud EU region
+
+The status page should now be available at [https://status.company.com](https://status.company.com).
```
Comment (otisg) [inline docs/synthetics/status-pages.md:41]: Not sure that we need to hyperlink this because this will never work.

**Review by otisg — CHANGES_REQUESTED** [review]
2 small things

**Review by adnanrahic — COMMENTED** [review]
Good to go after the fixes Otis mentioned.

**Pull request merged**